src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
This is the only concern - I don't think we should use unfetch as a polyfill, as it doesn't properly polyfill i.e. if someone includes our lib, and then includes their own `fetch` polyfill later on, that `fetch` polyfill will see ours and assume that native is already in place so won't bother loading itself in. The user will then be left with `unfetch` even though `unfetch` doesn't provide the entire polyfill. Instead, should use the [ponyfill](https://github.com/developit/unfetch#usage) version.